<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
<<<<<<< HEAD
<<<<<<< HEAD
    <height>762</height>
=======
    <height>753</height>
>>>>>>> a13668a1f5ef79e57ff1f1e8a8ddae5505f7751e
=======
    <height>753</height>
>>>>>>> a13668a1f5ef79e57ff1f1e8a8ddae5505f7751e
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="sourceImg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>531</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="msg">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>741</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGraphicsView" name="targetImg">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>531</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="msg_width">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>640</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="msg_height">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>640</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_width">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>图像宽度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_height">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>650</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>图像高度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>源图像</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>目标图像</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionClear"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>图像操作</string>
    </property>
    <addaction name="actionRot90"/>
    <addaction name="actionThreshold"/>
    <addaction name="actionGray"/>
    <addaction name="actionCanny"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>图像分割</string>
    </property>
    <addaction name="actionPSP"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>清空</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionRot90">
   <property name="text">
    <string>旋转90°</string>
   </property>
  </action>
  <action name="actionThreshold">
   <property name="text">
    <string>二值化</string>
   </property>
  </action>
  <action name="actionGray">
   <property name="text">
    <string>灰度化</string>
   </property>
  </action>
  <action name="actionCanny">
   <property name="text">
    <string>边缘提取</string>
   </property>
  </action>
  <action name="actionPSP">
   <property name="text">
    <string>深度监督PSP</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
